conda/faiss-gpu/meta.yaml

```diff
@@ -18,7 +18,6 @@ requirements:
     - python
     - numpy >=1.11
     - mkl
-    - cudatoolkit
```

Comment:
why did you remove dependency on cudatoolkit?
are you assuming users has correct version of CUDA installed on system?

Reply:
Mainly because we only have cudatoolkit8.0 in the channel. And if we want to install faiss-gpu with cuda90, it still installs cudatoolkit8.0 which is a bit weird.